Emby.Server.Implementations/Library/MediaSourceManager.cs

```diff
@@ -160,9 +160,9 @@ await item.RefreshMetadata(new MediaBrowser.Controller.Providers.MetadataRefresh
                 list.Add(source);
             }
 
-            foreach (var source in list)
+            if (user != null)
```

Comment:
I don't see how this change can fix the issue you link at.
I only see minor performance improvement (which is good but not what description says).

What am I missing here?

Reply:
That's because that's not the change that fixed the issue, I think you overlooked the other part of the diff. :p

Reply:
Lol, I saw second part but thought it was new field, but it was actually a missing initialization! :) sorry about the fuss